install/kubernetes/cilium/charts/preflight/values.yaml

```diff
@@ -10,4 +10,14 @@ tofqdnsPreCache: ""
 # By default we should always validate the installed CNPs before upgrading
 # Cilium. This will make sure the user will have the policies deployed in the
 # cluster with the right schema.
-validateCNPs: true
\ No newline at end of file
+validateCNPs: true
+
+# Node labels for pod assignment
+# ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+## Node tolerations for pod assignment on nodes with taints
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+##
+tolerations:
+- operator: Exists
```

Comment:
should we keep the same value here ?

Reply:
```
      tolerations:
        - effect: NoSchedule
          key: node.kubernetes.io/not-ready
        - effect: NoSchedule
          key: node-role.kubernetes.io/master
        - effect: NoSchedule
          key: node.cloudprovider.kubernetes.io/uninitialized
          value: "true"
        - key: CriticalAddonsOnly
          operator: "Exists"
```

Reply:
Yes, I wasn't sure if it was necessary? I am happy to put them back if you feel being more permissive could lead to some issues 🤔

Reply:
@mvisonneau I would also prefer keeping the values.

Reply:
I put them back! 👍 :)